src/Model/Post/Link.php

```diff
@@ -40,36 +40,44 @@ class Link
 	/**
 	 * Check if the link is stored
 	 *
-	 * @param int $uri_id
-	 * @param string $url
-	 * @return bool
+	 * @param int $uriId URI id
+	 * @param string $url URL
+	 * @return bool Whether record has been found
 	 */
-	public static function exists(int $uri_id, string $url)
+	public static function exists(int $uriId, string $url): bool
 	{
-		return DBA::exists('post-link', ['uri-id' => $uri_id, 'url' => $url]);
+		return DBA::exists('post-link', ['uri-id' => $uriId, 'url' => $url]);
 	}
 
-	public static function getByLink(int $uri_id, string $url, $size = '')
+	/**
+	 * Returns URL by URI id and other URL
+	 *
+	 * @param int $uriId URI id
+	 * @param string $url URL
+	 * @param string size Size
```

Comment:
Please tell me you didn't do this everywhere.
```suggestion
	 * @param int $uriId
	 * @param string $url
	 * @param string $size
```